packages/Button/stories/examples/Variations.js

```diff
@@ -191,6 +191,74 @@ const ExampleStory = () => (
         disabled Button.Link
       </Button.Link>
     </p>
+    <p>
+      <Button.Icon onClick={clickHandler}>
+        <InfoIcon />
+      </Button.Icon>
+      <Button.Icon onClick={clickHandler} isDisabled>
+        <InfoIcon />
+      </Button.Icon>
+      <Button.Icon onClick={clickHandler} kind="primary">
+        <InfoIcon />
+      </Button.Icon>
+      <Button.Icon onClick={clickHandler} kind="primary" isDisabled>
+        <InfoIcon />
+      </Button.Icon>
+      <Button.Icon onClick={clickHandler} kind="link">
+        <InfoIcon />
+      </Button.Icon>
+      <Button.Icon onClick={clickHandler} kind="link" isDisabled>
+        <InfoIcon />
+      </Button.Icon>
+      <Button.Icon onClick={clickHandler} kind="minor">
+        <InfoIcon />
+      </Button.Icon>
+      <Button.Icon onClick={clickHandler} kind="minor" isDisabled>
+        <InfoIcon />
+      </Button.Icon>
+      <Gap.Inline />
+      <Gap.Inline />
+      <Gap.Inline />
+      <Button.Icon
+        onClick={clickHandler}
+        kind="minor"
+        css={`
+          color: ${tokens.color.gold};
+        `}
+      >
+        <InfoIcon />
+      </Button.Icon>
+      <Button.Icon
+        onClick={clickHandler}
+        kind="minor"
+        css={`
+          color: ${tokens.color.gold};
+        `}
+        isDisabled
+      >
+        <InfoIcon />
+      </Button.Icon>
+      <Button.Icon onClick={clickHandler} kind="minor">
+        <InfoIcon
+          css={`
+            color: ${tokens.color.gold};
```

Comment:
Custom colour applied via css to `<Button>`'s `children` (ie the actual `<Icon>` component).